common/djangoapps/util/milestones_helpers.py

```diff
@@ -313,6 +314,18 @@ def remove_content_references(content_id):
     return milestones_api.remove_content_references(content_id)
 
 
+def any_unfulfilled_milestones(course_id, user_id):
+    """ Returns a boolean if user has any unfulfilled milestones """
+    from milestones.exceptions import InvalidMilestoneRelationshipTypeException
```

Comment:
I believe the solutions team needed to write their code so that when someone ran the edx code without the milestones feature, it wouldn't explode.